CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+- BugFix: Fixed incorrect formatting of code blocks in web help. [#535](https://github.com/zowe/imperative/issues/535)
+
 ## `4.10.2`
 - Fix vulnerabilities by updating marked
```

Comment:
Don't we have spaces between the version number and the list?
I know it doesn't really matter to GH though
(just trying to be annoying)  😋